apps/wallet/src/ui/app/pages/dapp-tx-approval/index.tsx

```diff
@@ -1,13 +1,16 @@
 // Copyright (c) Mysten Labs, Inc.
```

Comment:
We definitely need to move the components from this file to separate files, I think defining multiple component in file makes stuff confusing - just a thought not necessarily something to change in this PR

Reply:
I'd actually recommend doing some of this now, I think this is an opportune time to do it.

Reply:
this is done.